build/flatpak/element.desktop.in

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```

Comment:
The `build` directory is (confusingly) for electron-builder's config, so we should keep things unrelated to electron builder somewhere else. Probably just a top-level `flatpak` directory?